Sources/Logging/Logging.swift

```diff
@@ -156,32 +335,78 @@ public enum LoggingSystem {
 
+    /// A logging metadata value. `Logger.Metadata.Value` is string, array, and dictionary literal convertible.
     public enum MetadataValue {
+        /// A metadata value which is a `String`.
         case string(String)
+
+        /// A metadata value which is some `CustomStringConvertible`.
         case stringConvertible(CustomStringConvertible)
+
+        /// A metadata value which is a dictionary from `String` to `Logger.Metadata.Value`.
         case dictionary(Metadata)
+
+        /// A metadata value which is an array of `Logger.Metadata.Value`s.
         case array([Metadata.Value])
     }
 
+    /// The log level.
     public enum Level: Int {
+        /// The `.debug` log level, appropriate for messages that contain information normally of use only when
+        /// debugging a program.
         case debug = 7
+
+        /// The `.info` log level, appropriate for informational messages.
         case info = 6
+
+        /// The `.notice` log level, appropriate for conditions that are not error conditions, but that may require
+        /// special handling.
         case notice = 5
+
+        /// The `.warning` log level, appropriate for messages that are not error conditions, but more severe than
+        /// `.notice`.
         case warning = 4
+
+        /// The `.error` log level, appropriate for error conditions.
         case error = 3
+
+        /// The `.critical` log level, appropriate for criticial error conditions that usually require immediate
+        /// attention.
         case critical = 2
+
+        /// The `.alert` log level, appropriate for conditions that should be corrected immediately, such as a corrupted
+        /// system database.
         case alert = 1
+
+        /// The `.emergency` log level, appropriate for panic conditions.
         case emergency = 0
     }
 
+    /// Construct a `Logger` given a `label` identifying the creator of the `Logger`.
+    ///
+    /// The `label` should identify the creator of the `Logger`. This can be an application, a sub-system, or even
+    /// a datatype.
+    ///
+    /// - parameters:
+    ///     - label: An identifier for the creator of a `Logger`.
     public init(label: String) {
         self = LoggingSystem.lock.withReaderLock { Logger(label: label, LoggingSystem.factory(label)) }
     }
     
-    // this is to provide an escape hatch for situations one must use a custom factory instead of the gloabl one
-    // we do not expect this API to be used in normal circumstances, so if you find yourself using it make sure its for a good reason
+    /// Construct a `Logger` given a `label` identifying the creator of the `Logger` or a non-standard `LogHandler`.
+    ///
+    /// The `label` should identify the creator of the `Logger`. This can be an application, a sub-system, or even
+    /// a datatype.
+    ///
+    /// This initialiser provides an escape hatch in case the global default logging backend implementation (set up
```

Comment:
Let's stick to american spelling perhaps? (yeah I dislike it as well, though it is used through all [official docs](https://docs.swift.org/swift-book/LanguageGuide/Initialization.html) and we say `initialized` in the sources as well; _consistency is king_).

initialiser -> `initializer`